### ISSUE TITLE
Gaps between data-scroll-sections for horizontal scroll directions

### ISSUE THREAD

**Issue by aleksandrevic** [locomotivemtl/locomotive-scroll#317]
Hello 👋

**Describe the bug**
Consider a simple data-scroll-container without any particular styling, which comprises a number of equally large (e.g. 100vw, 100vh) data-scroll-sections, again without any particular styling. The Locomotive-Scroll object is created on the data-scroll-container and the direction is set to 'horizontal'. On Big Sur 11.4 and basically all Browsers (Safari, Chrome, FF) there are white-gaps of equal width between the data-scroll-sections. I added a background-colour to the data-scroll-container to check whether the gaps are associated to it: yes they are. When scrolling to the end (right edge), the last section is not displayed entirely due to the gaps.

If I create the same setup and set the direction to 'vertical', no gaps are displayed, everything is fine.

**To Reproduce**
`const locoScroll = new LocomotiveScroll({
	el: document.querySelector('[data-scroll-container]'),
	smooth: true,
	direction: 'horizontal',
});`

**Expected behavior**
A horizontally, smoothly scrollable container/website with X sections placed one to another without any gaps in-between.

**Screenshots**
<img width="721" alt="image" src="https://user-images.githubusercontent.com/33627935/124900462-603a4800-dfe1-11eb-8fb2-a314f1f7b87d.png">

**Desktop (please complete the following information):**
 - OS: Big Sur 11.4
 - Browser Safari, Chrome, FF
 - Version latest (updated with npm)

Thank you 👊

**Comment by Dushyant1295** [comment]
I Encountered same issue and solved with by giving negative margin

try   margin:-2px;

**Comment by aleksandrevic** [comment]
@Dushyant1295 Thanks! Since you've been working with horizontal scrolling, may I ask you another one: have you any solution for stable scrolling horizontal -> vertical section -> horizontal. I've solved it somehow with locomotive, but upon resizing the window, everything breaks. I posted on stackoverflow as well

[https://stackoverflow.com/questions/68242974/locomotive-scroll-switching-from-horizontal-to-vertical-and-back](url)

I would appreciate any hint!
